ObjectAL/ObjectAL/OALSimpleAudio.m

```diff
@@ -737,5 +738,23 @@ - (bool) suspended
 	return [OALAudioSession sharedInstance].suspended;
 }
 
+#pragma mark Suspend
+
+- (void) suspendAudio {
+    ALContext *alContext = [OpenALManager sharedInstance].currentContext;
+    ALCcontext *alcContext = alContext.context;
+
+    [ALWrapper makeContextCurrent:nil];
+    [ALWrapper suspendContext:alcContext];
+}
+
+- (void) unsuspendAudio {
+    ALContext *alContext = [OpenALManager sharedInstance].currentContext;
+    ALCcontext *alcContext = alContext.context;
```

Comment:
Maybe a private function to get the `ALCcontext`?

Reply:
The only problem with this is that it's really confusing whether you are talking about a `ALContext` or a `ALCcontext`
I think I'll just leave it as-is